### PR TITLE
Release of LTest for Frama-C 29.0 (Copper)

### PR DESCRIPTION
**Pull request by tmarti2**
Update for LAnnotate, LUncov and LReplay tools.

This update mainly updates version numbers and compatibility with Frama-C 29.0 (Copper).

### CODE REVIEW

File: packages/lreplay/lreplay.0.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+]
+license: "LGPL-2.1-only"
+homepage: "git+https://git.frama-c.com/pub/ltest/lreplay"
+dev-repo: "git+https://git.frama-c.com/pub/ltest/lreplay"
+bug-reports: "git+https://git.frama-c.com/pub/ltest/lreplay/-/issues"
+depends: [
+  "base-unix"
+  "ocaml" {>= "4.13.1"}
+  "dune" {>= "3.7" & != "3.13.0"}
+  "menhir" {build}
```
Comment (mseri) [inline packages/lreplay/lreplay.0.1.1/opam:29]:
```suggestion
  "menhir" {build & >= "20180528"}
```

Needs `--infer-write-query` option

**Comment by raphael-proust** [comment]
The windows CI fails because `pkg-config` is not found. You need to add a dependency to `conf-pkg-config` to resolve this issue.

**Comment by tmarti2** [comment]
From what I understand, the windows Ci fails, but succeed to install conf-pkg-config for LAnnotate : `- installed conf-pkg-config.3`.

This comes from frama-c dependancies, and `conf-zlib` has `depends: ["conf-pkg-config" {build}]`.
I don't think it has anything to do with LAnnotate/LUncov, this windows CI probably fails on Frama-C too.

**Comment by mseri** [comment]
Thanks

**Pull request merged**